tests/requests.rs

```diff
@@ -414,9 +443,8 @@ async fn osu_matches() -> Result<()> {
 }
 
 #[tokio::test]
-#[ignore = "requires OAuth to not throw an error"]
```

Comment:
I would rather keep it this way to enable quick manual testing by commenting this line.